package.json

```diff
@@ -39,12 +39,9 @@
     "@types/underscore.string": "*",
     "@types/urijs": "^1.15.38",
     "@types/uuid": "^3.0.0",
-    "@types/webpack": "4.41.10",
-    "awesome-typescript-loader": "^3.5.0",
     "babel-loader": "^8.1.0",
-    "better-npm-run": "0.0.15",
+    "better-npm-run": "0.1.1",
```

Comment:
It looks like you may have missed (or intentionally skipped) rehydrating the package-lock.json in WNPRC_Purchasing. I would recommend doing this as I had done in https://github.com/LabKey/wnprc-modules/pull/153. Once you have the correct Node/NPM version configured you can do:

```
> cd WNPRC_Purchasing
> rm -rf node_modules && rm package-lock.json && npm install --legacy-peer-deps
```

This should leave you with a freshly updated package-lock.json that you can commit.

Reply:
Thank you Nick, I did purposely did not rehydrate the package.json, I was hoping to do that after you look at the changes.

Reply:
Got it. Yes, take a look at my other comment. Once we get your version and commands sorted out then we can rehydrate for purchasing.